package.json

```diff
@@ -13,8 +13,10 @@
     "moment": "2.15.2",
     "object-sizeof": "1.1.1",
     "page-metadata-parser": "0.6.0",
+    "prop-types": "15.6.0",
     "react": "15.3.2",
```

Comment:
You'll need to update `react` here as well since it is used by node to generate the preloaded versions

Reply:
It's not obvious to me why having react use its own prop-types internally with react-intl using the new stuff would be a problem, since the prop-types themselves aren't (shouldn't be?) passed around.  That said, it's not an unreasonable request; I'll poke...